UnityProject/Assets/Scripts/Items/Implants/Organs/Bones.cs

```diff
@@ -10,27 +11,41 @@ public class Bones : BodyPartFunctionality
 
 		public float GenerationOvershoot = 1;
 
+		public HungerComponent HungerComponent;
```

Comment:
hungry bones?

Reply:
well it needs access because it consumes more nutriment when making blood